WebRTCAppEE/play.html

```diff
@@ -3,68 +3,42 @@
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 	<title>Ant Media Server WebRTC/HLS/DASH Player</title>
-	<script src="js/fetch.js"></script>
-	<script src="js/promise.min.js"></script>
-	<script src="js/external/adapter-latest.js"></script>
-
 	<link href="css/player.css" rel="stylesheet">
+
 </head>
 
-<body style="background-color: transparent;">
+<body>
 
 	<div id='video-overlay'>
-		<!-- <img src="images/loading.gif" alt="loading image" /> -->
+		<img src="images/loading.gif" alt="loading image" />
 	</div>
 
-	<div id="video_info" style="display: none">
-		Stream will start playing automatically
+	<div id="video_info">
+		Your live stream will play automatically as soon as it's available. <br />Get ready to be part of the action!
+		<!--
+		<div>
+			<img id="logo" src="images/logo.png" />
+		</div>
+		-->
 	</div>
 
-	<!-- HLS Player -->
 	<div id="video_container">
-		<video id="video-player" class="video-js vjs-default-skin vjs-big-play-centered" controls="hidden" preload="auto">
-			<p class="vjs-no-js">
-				To view this video please enable JavaScript, and consider upgrading
-				to a web browser that <a href="http://videojs.com/html5-video-support/" target="_blank">supports
-					HTML5 video</a>
-			</p>
-		</video>
+		<video id='video-player' class='video-js vjs-default-skin vjs-big-play-centered' controls playsinline></video>
 	</div>
 
-	<!-- WebRTC Player -->
-	
-
-	<video 
-		id="remoteVideo" playsinline
-		style="object-fit: fill;">
-	</video>
-
-
-
-	<div id="videoPausedOverlay" style="display: none;" >
-		<img id="pauseIcon" src="images/icons/pause.svg" style="z-index: 2; width: 15%; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);"/>
-		<!-- <span style="z-index: 2; position: absolute; top: 70%; left: 50%; transform: translate(-50%, -50%); color: white;"> Video Paused - Right Click => "Play" to unpause</span> -->
-		<div style="opacity:50%; background-color: transparent; z-index: 1; position: absolute; top: 0; left: 0; overflow: hidden; height: 100% !important; width: 100% !important; "> </div>
-	</div>
-
-
-	<!-- 360 player is added dynamically -->
-	
-
-	<div id="networkWarning"></div>
+	<div id="networkWarning">Your connection isn't fast enough to play this stream!</div>
```

Comment:
Looks like networkWarning is handled inside embedded-player, so should we remove this text?  Or is the warning appearing often no longer a problem?